### PR TITLE
Fix/request creating

### PR DESCRIPTION
**Pull request by BBCmisha**

### CODE REVIEW

File: src/views/RequestItem.vue [hunk 1]
```diff
@@ -56,27 +61,39 @@
...
-          <span class="info-table__row-title">Multiplier</span>
-          <span class="info-table__row-txt">
-            {{ requestCalldata.multiplier }}
-          </span>
-        </div>
+        <template
+          v-if="
+            typeof requestCalldata === 'object' &&
+            requestCalldata !== null &&
+            !Array.isArray(requestCalldata)
```
Comment (lukachi) [inline src/views/RequestItem.vue:68]: create computed

File: src/views/RequestItem.vue [hunk 2]
```diff
@@ -127,19 +148,39 @@ export default defineComponent({
         resolveTime.value = formatDate(resPacketData.resolveTime)
         requestTimeRange.value = formatDateDifference(resPacketData.requestTime)
         resolveTimeRange.value = formatDateDifference(resPacketData.resolveTime)
-        requestCalldata.value = obiCoin.decode(reqPacketData.calldata)
+        requestCalldata.value = await _decodeCallData(reqPacketData.calldata)
 
-        if (requestStatus.value == 1) {
-          requestRates.value = obiRates.decode(resPacketData.result)
+        if (requestStatus.value === ResolveStatus.RESOLVE_STATUS_SUCCESS) {
```
Comment (lukachi) [inline src/views/RequestItem.vue:153]: create computed property and replace in all component

File: src/components/modals/RequestFormModal.vue [hunk 3]
```diff
@@ -189,12 +217,64 @@ const RequestFormModal = defineComponent({
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.tooltip {
+  display: none;
+  position: absolute;
+  bottom: 130%;
+  right: -5px;
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/components/modals/RequestFormModal.vue:252]: done

File: src/components/modals/RequestFormModal.vue [hunk 4]
```diff
@@ -189,12 +217,64 @@ const RequestFormModal = defineComponent({
...
+  min-width: 15rem;
+  padding: 1.2rem 2.4rem;
+  background: var(--clr__tooltip-bg);
+  border-radius: 8px;
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/components/modals/RequestFormModal.vue:256]: done

File: src/components/modals/RequestFormModal.vue [hunk 5]
```diff
@@ -189,12 +217,64 @@ const RequestFormModal = defineComponent({
...
+  color: var(--clr__tooltip-text);
+
+  &:before {
+    content: '';
+    display: block;
+    width: 0.6rem;
+    height: 0.6rem;
+    position: absolute;
+    bottom: -0.3rem;
+    right: 10px;
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/components/modals/RequestFormModal.vue:269]: done

File: src/components/modals/OracleScriptFormModal.vue [hunk 6]
```diff
@@ -42,6 +43,20 @@
...
 
+          <div class="app-form__field">
+            <label class="app-form__field-lbl"> Schema </label>
+            <textarea
+              class="app-form__field-input app-form__field-textarea"
+              name="oracle-script-schema"
+              rows="2"
+              v-model="form.schema"
+              :disabled="isLoading"
+            ></textarea>
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/components/modals/OracleScriptFormModal.vue:54]: done

File: src/views/RequestItem.vue [hunk 7]
```diff
@@ -16,14 +16,19 @@
...
         <div class="info-table__row">
           <span class="info-table__row-title">Sender</span>
-          <TitledLink
-            class="info-table__row-link"
-            :text="String(requestData?.requestPacketData.clientId)"
-          />
+          <a
+            class="app-table__cell-txt app-table__link"
+            :href="`
+              ${API_CONFIG.odinScan}/account/${requestData?.requestPacketData.clientId}
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/views/RequestItem.vue:27]: done

File: src/views/Requests.vue [hunk 8]
```diff
@@ -41,13 +45,15 @@
...
             <div class="app-table__cell">
-              <span class="app-table__title">Client ID</span>
-              <span
-                class="app-table__cell-txt"
-                :title="item.requestPacketData.clientId.toString()"
+              <span class="app-table__title">Sender</span>
+              <a
+                class="app-table__cell-txt app-table__link"
+                :href="`
+                  ${API_CONFIG.odinScan}/account/${item.requestPacketData.clientId}
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/views/Requests.vue:52]: done

File: src/views/RequestItem.vue (hunk 1 above)
Comment (BBCmisha) [reply src/views/RequestItem.vue:68]: done

File: src/views/RequestItem.vue (hunk 2 above)
Comment (BBCmisha) [reply src/views/RequestItem.vue:153]: done

**Pull request merged**